module/enrichers.mjs

```diff
@@ -180,6 +180,77 @@ export async function enrichCheck(config, label, options) {
 
 /* -------------------------------------------- */
 
+/**
+ * Enrich a damage link.
+ * @param {string[]} config            Configuration data.
+ * @param {string} label               Optional label to replace default text.
+ * @param {EnrichmentOptions} options  Options provided to customize text enrichment.
+ * @returns {HTMLElement|null}         An HTML link if the save could be built, otherwise null.
+ *
+ * @example Create a damage link:
+ * ```[[/damage 2d6 type=bludgeoning]]``
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="damage" data-formula="2d6" data-damage-type="bludgeoning">
+ *   <i class="fa-solid fa-dice-d20"></i> 2d6
+ * </a> bludgeoning
+ * ````
+ *
+ * @example Display the average:
+ * ```[[/damage 2d6 type=bludgeoning average=true]]``
+ * becomes
+ * ```html
+ * 7 (<a class="roll-action" data-type="damage" data-formula="2d6" data-damage-type="bludgeoning">
+ *   <i class="fa-solid fa-dice-d20"></i> 2d6
+ * </a>) bludgeoning
+ * ````
+ *
+ * @example Manually set the average & don't prefix the type:
+ * ```[[/damage 8d4dl force average=666]]``
+ * becomes
+ * ```html
+ * 666 (<a class="roll-action" data-type="damage" data-formula="8d4dl" data-damage-type="force">
+ *   <i class="fa-solid fa-dice-d20"></i> 8d4dl
+ * </a> force
+ * ````
+ */
+export async function enrichDamage(config, label, options) {
+  const formulaParts = [];
+  if ( config.formula ) formulaParts.push(config.formula);
+  for ( const value of config.values ) {
+    if ( value in CONFIG.DND5E.damageTypes ) config.type = value;
```

Comment:
We could add our well-known boolean values here to allow them to be passed in as 'flag-style' still:

```suggestion
    if ( value in CONFIG.DND5E.damageTypes ) config.type = value;
    else if ( value === "average" ) config.average = true;
```

Reply:
I was thinking just parsing any unrecognized values as flags:

```suggestion
    if ( value in CONFIG.DND5E.damageTypes ) config.type = value;
    else config[value] = true;
```